azure-templates/steps-diff.yml

```diff
@@ -0,0 +1,56 @@
+parameters:
+  - name: lvVersionToDiff
+    type: string
+
+# Note: the following variables are defined in pre-job-steps.yml:
+  # customDeviceRepoName
+
+steps:
+  - task: PowerShell@2
+    displayName: Prepare diff environment
+    inputs:
+      script: |
+        pip install virtualenv
+        virtualenv env
+        .\env\Scripts\activate
+        pip install requests
+  - checkout: self
+    fetchDepth: 0
+    displayName: Fetch full repo
+  - task: PowerShell@2
```

Comment:
Is there a reason that this is a powershell task instead of a - [checkout](https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-checkout?view=azure-pipelines): step?